src/bp/core/services/services.inversify.ts

```diff
@@ -45,7 +45,7 @@ const ServicesContainerModule = new ContainerModule((bind: interfaces.Bind) => {
   bind<JobService>(TYPES.JobService)
     .to(CEJobService)
     .inSingletonScope()
-    .when(() => !process.IS_PRO_ENABLED)
+    .when(() => !process.IS_REDIS_ENABLED)
```

Comment:
I don't mind if we leave it this way, I'm simply sharing my thoughts here, I feel like its always better to avoid naming the tech we use for application purposes bc if we ever happen to change or make messaging tech configurable we'll need to refact this... 

you could rename this for smth lime `CLUSTER_ENABLED` or `IS_CLUSTER_MODE`

Reply:
Good point